src/sentry/integrations/opsgenie/integration.py

```diff
@@ -120,7 +121,46 @@ def dispatch(self, request: Request, pipeline) -> HttpResponse:  # type:ignore
 
 
 class OpsgenieIntegration(IntegrationInstallation):
-    pass
+    def get_client(self) -> Any:
+        org_integration_id = self.org_integration.id if self.org_integration else None
+        return OpsgenieClient(integration=self.model, org_integration_id=org_integration_id)
+
+    def get_organization_config(self) -> Sequence[Any]:
+        fields = [
+            {
+                "name": "team_table",
+                "type": "table",
+                "label": "Opsgenie teams with the Sentry integration enabled",
+                "help": "If teams need to be updated, deleted, or added manually please do so here. Alert rules will need to be individually updated for any additions or deletions of teams.",
```

Comment:
nit: this is pretty subjective but I prefer a more concise message
"Manually add, update, or delete teams here. Alert rules must be updated individually for team additions and deletions.

Reply:
I'm using the one from PagerDuty, so to keep things consistent I'll probably keep it.